docs/io.cozy.home.settings.md

```diff
@@ -35,8 +35,8 @@ The `io.cozy.home.settings` doctype contains one document that manages settings
           - **Description**: If `true`, the mobile view will display shortcuts with detailed lines.
         - **`grouped`**: {boolean}
           - **Description**: If `true`, shortcuts are grouped together on the mobile view.
-    - **`order`**: {null}
-      - **Description**: Placeholder for future order-related information; currently `null`.
+    - **`order`**: {number}
+      - **Description**: If provided, will display shortcuts from lowest to highest number (default is `infinity`).
```

Comment:
I see the whole io.cozy.home.settings doctype is formatted like this, but you don't need to add a `description` section below the attribute. Everywhere else it's more like this: 
` order: {number} If provided, will display shortcuts from lowest to highest number (default is `infinity`).`

Reply:
will change